### PR TITLE
store/tikv: mock tikv tests

### PR DESCRIPTION
**Pull request by disksing**
@shenli @coocood @morefreeze

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: store/tikv/mock-tikv/cluster_manipulate.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mocktikv
+
+import "fmt"
+
+// BootstrapWithSingleStore initializes a Cluster with 1 Region and 1 Store.
+func BootstrapWithSingleStore(cluster *Cluster) {
```
Comment (coocood) [inline store/tikv/mock-tikv/cluster_manipulate.go]: Why not define this function in `cluster.go`?

Comment (disksing) [inline store/tikv/mock-tikv/cluster_manipulate.go]: @coocood A lot more functions (LeaderChange, KillNode, etc.) will be added later. They are considered as more higher level batch operations applied on the Cluster.

**Comment by coocood** [comment]
LGTM

**Pull request merged**